moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -169,7 +169,15 @@ class TimeOptimalTrajectoryGeneration : public TimeParameterization
   bool computeTimeStamps(robot_trajectory::RobotTrajectory& trajectory, const double max_velocity_scaling_factor = 1.0,
                          const double max_acceleration_scaling_factor = 1.0) const override;
 
+  bool computeTimeStamps(robot_trajectory::RobotTrajectory& trajectory,
```

Comment:
Would it make sense to add this overload to the base class?

Reply:
@JafarAbdi I went ahead and added it to the base class. Can you give the new commit a review?  Thanks